tools/travis-setup-rmq.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
+source tools/travis-common-vars.sh
+NAME=$(db_name rmq)
+RMQ_PORT=5672
+
+docker rm -v -f $NAME || echo "Skip removing the previous container"
 docker run -d \
-       --name mongooseim-rmq \
-       -p 5672:5672 \
+       --name $NAME \
+       -p $RMQ_PORT:$RMQ_PORT \
```

Comment:
The inner port is fixed by the container, wouldn't make any sense to be able to change it here, this pattern is usually so that we can map to a chosen port on the host, not on the container.
```suggestion
       -p $RMQ_PORT:5672 \
```

Reply:
the first port is the host port
https://docs.docker.com/engine/reference/run/#expose-incoming-ports

Reply:
sorry i'm "blind", I misunderstood the comment and somehow skipped the suggestion. you're right